src/WebJobs.Script/Binding/ServiceBusScriptBindingProvider.cs

```diff
@@ -65,29 +65,7 @@ public override void Initialize()
                 }
             }
 
-            EventProcessorOptions eventProcessorOptions = EventProcessorOptions.DefaultOptions;
-            eventProcessorOptions.MaxBatchSize = 1000;
-            int batchCheckpointFrequency = 1;
-            configSection = (JObject)Metadata.GetValue("eventHub", StringComparison.OrdinalIgnoreCase);
-            if (configSection != null)
-            {
-                if (configSection.TryGetValue("maxBatchSize", StringComparison.OrdinalIgnoreCase, out value))
-                {
-                    eventProcessorOptions.MaxBatchSize = (int)value;
-                }
-
-                if (configSection.TryGetValue("prefetchCount", StringComparison.OrdinalIgnoreCase, out value))
-                {
-                    eventProcessorOptions.PrefetchCount = (int)value;
-                }
-
-                if (configSection.TryGetValue("batchCheckpointFrequency", StringComparison.OrdinalIgnoreCase, out value))
-                {
-                    batchCheckpointFrequency = (int)value;
-                }
-            }
-            _eventHubConfiguration = new EventHubConfiguration(eventProcessorOptions);
-            _eventHubConfiguration.BatchCheckpointFrequency = batchCheckpointFrequency;
+            _eventHubConfiguration = new EventHubConfiguration();
 
             Config.UseServiceBus(serviceBusConfig);
             Config.UseEventHub(_eventHubConfiguration);
```

Comment:
Should just be able to call Config.UseEventHub(). I sent a PR adding that overload: https://github.com/Azure/azure-webjobs-sdk/pull/1684

Reply:
I merged that change and pulled new version into Functions repo.